documentation/long-stay-baseline.mdx

```diff
@@ -0,0 +1,528 @@
+
+Additionally, Length of Stay was capped to 30 days, due to a long tail of long stayers over ~15 days and the definition of long stayer being over 21 days. The effect of capping can be visualised by comparing box plots of the distribution of length of stay on the raw data (left image, y scale up to 300 days) and the capped data (right image, y scale up to 30 days):
+
+<p align="center">![Boxplot of length of stay](images/long-stay-baseline/los-boxplot.png)</p>
+
+The resulting distribution of length of stays shows a ~bimodal distribution - the majority of stays are short (&lt;5 days), which a long tail and population of long stayers:
+
+<p align="center">![Density plot of length of stay](images/long-stay-baseline/los-density.jpeg)</p>
+
+## 4. Feature engineering<a name='featureengineering'></a>
+
+After discussion with GHNHSFT, the following decisions were made in feature selection:
+
+1. Select the following features for inclusion in the model, which are available on admission:
+```python
+    "ae_arrival_mode",
+    "AGE_ON_ADMISSION",
+    "EL CountLast12m",
+    "EMCountLast12m",
+    "IS_illness_not_injury",
+    "IS_cancer",
+    "IS_care_home_on_admission",
+    "IS_chronic_kidney_disease",
+    "IS_COPD",
+    "IS_coronary_heart_disease",
+    "IS_dementia",
+    "IS_diabetes",
+    "IS_frailty_proxy",
+    "IS_hypertension",
+    "IS_mental_health",
+    "MAIN_SPECIALTY_CODE_AT_ADMISSION_DESCRIPTION",
+    "OP First CountLast12m",
+    "OP FU CountLast12m",
+    "SOURCE_OF_ADMISSION_HOSPITAL_PROVIDER_SPELL_DESCRIPTION",
+    "stroke_ward_stay",
+    "LENGTH_OF_STAY",
+    "arrival_day_of_week",
+    "arrival_month_name"
+```
+2. Exclude the following features, but retain for later analysis of model fairness:
+```python
+    "ETHNIC_CATEGORY_CODE_DESCRIPTION",
+    "IMD county decile",
+    "OAC Group Name",
+    "OAC Subgroup Name",
+    "OAC Supergroup Name",
+    "PATIENT_GENDER_CURRENT_DESCRIPTION",
+    "POST_CODE_AT_ADMISSION_DATE_DISTRICT",
+    "Rural urban classification"
+```
+3. Generate `arrival_day_of_week` and `arrival_month_name` recalculated from `START_DATE_TIME_HOSPITAL_PROVIDER_SPELL`
+
+This resulted in a dataset of **~170,000 rows across 30 columns**.
+
+One-hot encoding was performed for categorical variables, but non-one-hot encoded features were also kept for models like CatBoost which [manage categorical variables themselves](https://catboost.ai/en/docs/features/categorical-features).
+
+### 5. Statistical analysis<a name='statisticalanalysis'></a>
+
+In order to select appropriate modelling approaches, some basic statistical analysis was conducted to understand normality and inter-correlation of the selected features.
+
+#### Correlation analysis
+
+Correlation analysis confirmed presence of significant **collinearity** between different features.
+
+Top 20 one-hot encoded features correlated with `LENGTH_OF_STAY`, ranked by absolute correlation, were:
+
+Feature|Correlation (Pearson correlation coefficient)
```

Comment:
Is it possible to show in a histogram plot and/or list them in descending order w.r.t. the `Correlation (Pearson correlation coefficient)`?